ROADMAP.md

```diff
@@ -1,91 +1,108 @@
 Roadmap
 -------
 
-## Goal
+The Data Object Service has been implemented in multiple workflow execution engines 
+and has helped to allow cloud agnostic data access. A small network of DOS servers 
+is available from a resolver at dataguids.org.
 
-Create an interoperability layer that allows metadata about resources to be interchanged using a common client.
+Software for presenting Human Cell Atlas, NCI GDC, GTEx, and TOPMed data are all made 
+available. Making it easier to discover, interact with, access, and publish new data 
+using supporting software, documentation, and possible schema changes make up the 
+bulk of this roadmap.
 
-## Metrics of success
+## Goal
 
-What data are available by DOS? GDC? TOPMed? HCA? How many files/bundles/PB?
+### Short term - 6 months
 
-How many workflows have been converted to use DOS downloader?
+Release version 1 and improve ease of use and adoption through supporting software. 
```

Comment:
@david4096 Considering the cloud workstream hasn't even started discussing and/or developing DOS, doesn't this seem like an aggressive timeline?

Reply:
Although it's hardly the case the workstream hasn't discussed DOS, this release candidate would be the one we would move forward with the approval process on, I presume. @briandoconnor  ?